docs/pages/guides/extending-world.mdx

```diff
@@ -484,3 +488,567 @@ The easiest way to create the Solidity code is to use the MUD template:
+Here we modify the template UI to so it will run properly.
+
+1. Change to the client package of the extension.
+
+   ```sh copy
+   cd ../client
+   ```
+
+1. Port 3000 is already in use by the `extendMe` application, so edit `vite.config.ts` to use a different port.
+
+   <CollapseCode>
+
+   ```ts filename="vite.config.ts" copy showLineNumbers {5}
+   import { defineConfig } from "vite";
+
+   export default defineConfig({
+     server: {
+       port: 3001,
+       fs: {
+         strict: false,
+       },
+     },
+     build: {
+       target: "es2022",
+       minify: true,
+       sourcemap: true,
+     },
+   });
+   ```
+
+   </CollapseCode>
+
+1. Start the user interface.
+
+   ```sh copy
+   pnpm vite
+   ```
+
+1. Open the application at [http://localhost:3001](http://localhost:3001).
+   At this point the server gives us web pages, but the JavaScript in them does not work, the next steps are to fix it.
+
+1. Because `Counter` is not defined in the extension's `mud.config.ts`, it isn't one of the components.
+   Comment out the lines that register a handled for updates on `components.Counter` in `src/index.ts`.
+
+   <CollapseCode>
+
+   ```ts filename="index.ts" copy showLineNumbers {10-17}
+   import { setup } from "./mud/setup";
+   import mudConfig from "contracts/mud.config";
+
+   const {
+     components,
+     systemCalls: { increment },
+     network,
+   } = await setup();
+
+   /*
+   // Components expose a stream that triggers when the component is updated.
+   components.Counter.update$.subscribe((update) => {
+     const [nextValue, prevValue] = update.value;
+     console.log("Counter updated", update, { nextValue, prevValue });
+     document.getElementById("counter")!.innerHTML = String(nextValue?.value ?? "unset");
+   });
+   */
+
+   // Just for demonstration purposes: we create a global function that can be
+   // called to invoke the Increment system contract via the world. (See IncrementSystem.sol.)
+   (window as any).increment = async () => {
+     console.log("new counter value:", await increment());
+   };
+
+   // https://vitejs.dev/guide/env-and-mode.html
+   if (import.meta.env.DEV) {
+     const { mount: mountDevTools } = await import("@latticexyz/dev-tools");
+     mountDevTools({
+       config: mudConfig,
+       publicClient: network.publicClient,
+       walletClient: network.walletClient,
+       latestBlock$: network.latestBlock$,
+       storedBlockLogs$: network.storedBlockLogs$,
+       worldAddress: network.worldContract.address,
+       worldAbi: network.worldContract.abi,
+       write$: network.write$,
+       recsWorld: network.world,
+     });
+   }
+   ```
+
+   </CollapseCode>
+
+1. The network configuration does not have the correct `World` address.
+   Get the `World` address from the application being extended.
+   By default, you can do see this application on URL [http://localhost:3000](http://localhost:3000).
+   Edit `src/mud/getNetworkConfig.ts` to set the `World` address.
```

Comment:
Why do we have to edit `getNetworkConfig`? Shouldn't it be sufficient to update the `worlds.json` file?